pyston/resource.py

```diff
@@ -205,6 +205,7 @@ class BaseResource(PermissionsResourceMixin, metaclass=ResourceMetaClass):
     converter_classes = settings.CONVERTERS
     errors_response_class = settings.ERRORS_RESPONSE_CLASS
     error_response_class = settings.ERROR_RESPONSE_CLASS
+    field_labels = {}
```

Comment:
because of csv/xlsx exports the field_labels can be changed in resource (similary like in is-core)